base/applications/games/solitaire/solitaire.cpp

```diff
@@ -738,7 +739,8 @@ LRESULT CALLBACK WndProc (HWND hwnd, UINT iMsg, WPARAM wParam, LPARAM lParam)
                 return 0;
 
             case IDM_HELP_ABOUT:
-                MessageBox(hwnd, MsgAbout, szAppName, MB_OK|MB_ICONINFORMATION);
+                ShellAboutW(hwnd, szAppName, MsgAbout,
+                            LoadIconW(hInstance, MAKEINTRESOURCEW(IDI_SOLITAIRE)));
```

Comment:
Ah, (this was a remark from @SergeGautherie ) since the code is using TCHARs for string buffers, then it's better to actually not use the explicit unicode functions here.
```suggestion
                ShellAbout(hwnd, szAppName, MsgAbout,
                           LoadIcon(hInstance, MAKEINTRESOURCE(IDI_SOLITAIRE)));
```